react/features/breakout-rooms/actions.js

```diff
@@ -233,11 +233,12 @@ export function moveToRoom(roomId?: string) {
             const isVideoMuted = isLocalCameraTrackMuted(localTracks);
 
             try {
-                await APP.conference.leaveRoom(false /* doDisconnect */);
+                await APP.conference.leaveRoom(false /* doDisconnect */).then(
+                    () => APP.API.notifyConferenceLeft(APP.conference.roomName));
```

Comment:
Could you please add a comment why do we use `APP.conference.roomName` instead of something from redux?